Adding API AM production warnings to relevant pages (#2849)
okta/okta-developer-docs: packages/@okta/vuepress-site/docs/concepts/auth-servers/index.md

  - name: description
    content: An overview of what an authorization server is and the types of authorization servers available at Okta.
---

# Authorization Servers

> > **Note**: [Okta’s Developer Edition](/signup/) makes most key developer features available by default for testing purposes. Please note that [API Access Management](/docs/concepts/api-access-management/) — a requirement for Custom Authorization Servers — is an optional add-on in production.

Reviewer: ```suggestion
> **Note**: [Okta’s Developer Edition](/signup/) makes most key developer features available by default for testing purposes. [API Access Management](/docs/concepts/api-access-management/) — a requirement for Custom Authorization Servers — is an optional add-on for production environments.
```

Author: Wording updated; thanks!
